src/client/features/interactions/filter-menu.js

```diff
@@ -0,0 +1,19 @@
+const React = require('react');
+const h = require('react-hyperscript');
+const classNames = require('classnames');
+class InteractionsFilterMenu extends React.Component {
+  render(){
+   const props= this.props;
+   const buttons= [...props.buttons].map(([type, clicked])=>
+   h('div',{key:type,className:classNames ('interaction-filter-button',clicked? 'interaction-filter-clicked':'interaction-filter-not-clicked'),onClick: () => props.filterUpdate(type)},[
```

Comment:
Split these up for readability

`'a', 'b', { 'c': condition }`

```
{
  'a': true,
  'b': true,
  'c': condition
}
```

The lines are long. Use newlines on your objects where it makes sense